codebase-pipelines/buildspec.yml

```diff
@@ -0,0 +1,48 @@
+version: 0.2
+
+env:
+  parameter-store:
+    SLACK_CHANNEL_ID: /codebuild/slack_oauth_channel
+    SLACK_TOKEN: /codebuild/slack_oauth_token
+
+phases:
+  install:
+    commands:
+      - >
+        if [ -f .copilot/phases/install.sh ]; then 
+          bash .copilot/phases/install.sh; 
+        fi
+
+  pre_build:
+    commands:
+      - >
+        if [ -f .copilot/phases/pre_build.sh ]; then 
+          bash .copilot/phases/pre_build.sh; 
+        fi
+
+  build:
+    commands:
+      - >
+        if [ -f .copilot/phases/build.sh ]; then 
+          bash .copilot/phases/build.sh; 
+        fi
+      - /work/cli build --publish --send-notifications
+
+  post_build:
+    commands:
+      - >
+        if [ -f .copilot/phases/post_build.sh ]; then 
+          bash .copilot/phases/post_build.sh; 
+        fi
+        
+        if [ "${CODEBUILD_BUILD_SUCCEEDING}" != "1" ]; then
+          BUILD_ID_PREFIX=$(echo $CODEBUILD_BUILD_ID | cut -d':' -f1)
+          echo "BUILD_ID_PREFIX - ${BUILD_ID_PREFIX}"
+
+          echo -e "\nInstalling dependencies"
+          pip install dbt-platform-helper
```

Comment:
How do we feel about pre-installing this on `ci-image-builder`?

We would get the notifications 30 to 60 seconds earlier and it would be MUCH easier to find the reason the run failed in the CodeBuild output.

Reply:
It's saving the build time vs having to update the `platform-helper` version install in `ci-image-builder`, I think it's probably worth it.

We could reduce the log noise regardless with a quiet flag on the install.

Reply:
I don't think it matters if the version baked into the image isn't the latest, as long as the `notify` commands work.

Reply:
I'll add a ticket for it